content/en/synthetics/mobile_app_testing/mobile_app_tests/steps.md

```diff
@@ -309,11 +309,21 @@ After adding the subtest to your mobile test, click the subtest to view another
 
 {{< img src="mobile_app_testing/test_steps/subtest_preview_steps_click.png" alt="Selecting a subtest shows a preview of the steps" style="width:60%" >}}
 
+## Variables
 
-### Variables
-If your subtest contains variables, they are inherited by the test you import them into. 
+{{% synthetics-variables %}}
+
+**Note**: If your subtest contains variables, they are inherited by the test you import them into. 
 To override these variables, create a variable in your parent test with the name as the variables within your subtest. 
 
+### Use global variables
+
+You can use the [global variables defined in **Settings**][4] in the **Advanced Options** of your mobile app test details, as well as in your test recording to define local variables. To display a list of available variables, type `{{` in the desired field.
```

Comment:
Optional suggestions, from a user perspective, view might be more accurate than display?

```suggestion
You can use the [global variables defined in **Settings**][4] in the **Advanced Options** of your mobile app test details, as well as in your test recording to define local variables. To view a list of available variables, type `{{` in the desired field.
```